### PR TITLE
[5.3] Add option for show/hide title in readmore

### PR DESCRIPTION
**Pull request by rdeutz**
Pull Request for Issue #34609 .

### Summary of Changes
Adding an option to the newsflash module to show, hide use global setting for the text at the read more button

### Testing Instructions
1) Read the issue and confirm the issue on Joomla 5.2 with sample data
2) apply patch
3) go to the "Latest Posts" module
4) change the "Read More with Title" options in the module and check if the hide/show have the effect we want
5) change the "Read More with Title" options in the module to "Use global article settings"
6) go to content -> options and change under articles the Read More with Title and check if the hide/show have the effect we want

### Actual result BEFORE applying this Pull Request
See issue

### Expected result AFTER applying this Pull Request
You can control from the newflash module what text is at the read more button in the newsflash module

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: language/en-GB/mod_articles_news.ini [hunk 1]
```diff
@@ -18,6 +18,7 @@ MOD_ARTICLES_NEWS_FIELD_ORDERING_ORDERING="Ordering"
 MOD_ARTICLES_NEWS_FIELD_ORDERING_PUBLISHED_DATE="Published Date"
 MOD_ARTICLES_NEWS_FIELD_ORDERING_RANDOM="Random"
 MOD_ARTICLES_NEWS_FIELD_READMORE_LABEL="'Read more &hellip;' Link"
+MOD_ARTICLES_NEWS_FIELD_READMORE_TITLE_LABEL="Read More with Title"
```
Comment (brianteeman) [inline language/en-GB/mod_articles_news.ini]: Why add a new string when we already have a global string
JGLOBAL_SHOW_READMORE_TITLE_LABEL

File: modules/mod_articles_news/mod_articles_news.xml [hunk 2]
```diff
@@ -160,6 +160,18 @@
 					<option value="1">JSHOW</option>
 				</field>
 
+				<field
+					name="readmore_title"
+					type="list"
+					label="MOD_ARTICLES_NEWS_FIELD_READMORE_TITLE_LABEL"
+					default=""
+					showon="readmore:1"
+				>
```
Comment (brianteeman) [inline modules/mod_articles_news/mod_articles_news.xml]:
```suggestion
					>
```

Comment (sandewt) [reply modules/mod_articles_news/mod_articles_news.xml]: Add `validate="options"` see eg line 191 as an example

File: language/en-GB/mod_articles_news.ini (hunk 1 above)
Comment (rdeutz) [reply language/en-GB/mod_articles_news.ini]: MIssed that we have a tag

File: modules/mod_articles_news/mod_articles_news.xml (hunk 2 above)
Comment (rdeutz) [reply modules/mod_articles_news/mod_articles_news.xml]: done

**Comment by KingLouis1** [comment]
I have not tested this item.<br /><br />Not able to test, as instruction not clear. Where ist this module?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by crimle** [comment]
I am bewildered. First you mention the «newsflash module». Later you ask «go to the "Latest Posts" module». I was unable to test thoroughly.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

File: modules/mod_articles_news/mod_articles_news.xml [hunk 3]
```diff
@@ -160,6 +160,19 @@
...
 
+				<field
+					name="readmore_title"
+					type="list"
+					label="JGLOBAL_SHOW_READMORE_TITLE_LABEL"
+					default=""
+					showon="readmore:1"
+					validate="options"
+					>
+					<option value="">MOD_ARTICLES_NEWS_VALUE_USE_ARTICLE_SETTINGS</option>
```
Comment (Quy) [inline modules/mod_articles_news/mod_articles_news.xml]:
```suggestion
					<option value="">JGLOBAL_USE_GLOBAL</option>
```

To be consistent with another setting on the same page.

43457-linked-titles

File: language/en-GB/mod_articles_news.ini [hunk 4]
```diff
@@ -28,4 +28,5 @@ MOD_ARTICLES_NEWS_READMORE="Read more &hellip;"
 MOD_ARTICLES_NEWS_READMORE_REGISTER="Register to Read More"
 MOD_ARTICLES_NEWS_TITLE_HEADING="Header Level"
 MOD_ARTICLES_NEWS_VALUE_ONLY_SHOW_FEATURED="Only show Featured Articles"
+MOD_ARTICLES_NEWS_VALUE_USE_ARTICLE_SETTINGS="Use Global Article Settings"
```
Comment (Quy) [inline language/en-GB/mod_articles_news.ini]:
```suggestion
```

**Comment by crimle** [comment]
I have tested this item :white_check_mark: successfully on 0a2af87b17da339c384b8f77e14c746f84c9a9b9<br /><br />Should be
3. go to the "**Articles - Newsflash**" module
NOT 3. go to the "Latest Posts" module
then the testing instructions are comprehensible. My tests were successful. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 0a2af87b17da339c384b8f77e14c746f84c9a9b9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by richard67** [comment]
@rdeutz Could you check the changes suggested by @Quy ?

**Comment by pcombet-adosis** [comment]
Hello, 
tested, it does work ✅
I have tested it in a Multi language environment (FR/ENG)
All clear
Best 
Phil

**Comment by richard67** [comment]
> Hello, tested, it does work ✅ I have tested it in a Multi language environment (FR/ENG) All clear Best Phil

@pcombet-adosis Please go to the issue tracker here https://issues.joomla.org/tracker/joomla-cms/43457 and mark your test result by using the blue "Test this" button at the top left corner, selecting your test result and finally submit, so that your test is properly counted. Just a comment with a green check mark is not enough. Thanks in advance.

**Comment by pcombet-adosis** [comment]
I have tested this item :white_check_mark: successfully on 0a2af87b17da339c384b8f77e14c746f84c9a9b9<br /><br />I have tested it in a Multi language environment (FR/ENG)
All clear

Best 
Philippe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by crommie** [comment]
I have tested this item :white_check_mark: successfully on 0a2af87b17da339c384b8f77e14c746f84c9a9b9<br /><br />Works as described.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by softforge** [comment]
I have tested this item :white_check_mark: successfully on 0a2af87b17da339c384b8f77e14c746f84c9a9b9<br /><br />Did what it says on the tin<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by richard67** [comment]
As the last change after the successful human tests did just change the language string to the already existing global string, which is already used elsewhere in the same XML, which I've just checked by review, the previous human tests are still valid => RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43457">issues.joomla.org/tracker/joomla-cms/43457</a>.</sub>

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.3-dev.

**Pull request merged**